graphql/mod.ts

```diff
@@ -0,0 +1,67 @@
+import { Drash } from "../deps.ts";
+import {
+  ExecutionResult,
+  graphql,
+  GraphQLSchema,
+  renderPlaygroundPage,
+} from "./deps.ts";
+
+interface GraphQLOptions {
+  schema: GraphQLSchema;
+  graphiql: boolean | string;
+  rootValue: Record<string, () => string>;
+}
+
+/**
+ * Taken from https://github.com/deno-libs/gql/blob/master/http.ts but heavily modified to suit drash's needs,
+ * and to utilise as much of graphql's own code instead
+ */
+export function GraphQL(
+  options: GraphQLOptions,
+): (
+  request: Drash.Http.Request,
+  response: Drash.Http.Response,
+) => Promise<Drash.Http.Response> {
+  return async (request: Drash.Http.Request, response: Drash.Http.Response) => {
+    const playgroundEndpoint = options.graphiql === true
+      ? "/graphql"
+      : typeof options.graphiql === "string"
+      ? options.graphiql
+      : undefined;
+    if (
+      playgroundEndpoint && request.method === "GET" &&
+      (request.headers.get("Accept")?.includes("text/html") ||
+        request.headers.get("Accept")?.includes("*/*"))
+    ) {
+      response.headers.set("Content-Type", "text/html");
+      response.body = renderPlaygroundPage({ endpoint: playgroundEndpoint });
+      return response;
+    }
+
+    if (!["PUT", "POST", "PATCH"].includes(request.method)) {
```

Comment:
why not `else` here instead? is `DELETE` allowed?

Reply:
oh i see this is not _just_ for the playground.

Reply:
What you said earlier is making me think... i know deno's gql module does this, but you said you use GETs for querying your graphql server, so maybe we should allow gets? And maybe it's possible to separate these two bits of functionality?

Reply:
tbh i think this is a great start! would be nice to put lines 47 - 62 in another func... `executeRequest(request)`  or something...?

Reply:
Sgtm, what about this: https://github.com/drashland/deno-drash-middleware/pull/118/commits/ded9563a2fce824009dfa50ac1ec04f069e71a30

Reply:
😍

Reply:
XD I know you love abstracting everything into functions lol